src/index.scss

```diff
@@ -0,0 +1,6 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+@import './src/stylings/re-usable/reset.scss';
+@import './src/stylings/re-usable/navbar.scss';
```

Comment:
use camelCase for file naming for consistency

Reply:
This has been resolved